docs/developer-guide/nodejs-api.md

```diff
@@ -49,15 +49,15 @@ var codeLines = SourceCode.splitLines(code);
  */
 ```
 
-## linter
+## Linter
 
-The `linter` object does the actual evaluation of the JavaScript code. It doesn't do any filesystem operations, it simply parses and reports on the code. In particular, the `linter` object does not process configuration objects or files. You can retrieve `linter` like this:
+The `Linter` object does the actual evaluation of the JavaScript code. It doesn't do any filesystem operations, it simply parses and reports on the code. In particular, the `linter` object does not process configuration objects or files. You can retrieve `linter` like this:
```

Comment:
Nitpick: There are two references to `linter` on this line that should be changed to `Linter`.

Also, it might be a good idea to clarify that `Linter` is a constructor. For example, you could say "You can retrieve instances of `Linter` like this:"